### ISSUE TITLE
Convert docs to sphinx and put on RTFD.org

### ISSUE THREAD

**Issue by pydanny** [GoodCloud/django-zebra#6]
Another thing that will increase usage is Sphinx for better documentation. And hosting of those docs on rtfd.org

**Comment by skoczen** [comment]
Yup. RTD is coming - I've actually got some stuff I told eric I'd work on on that project, so will probably use zebra to test out those changes.

Again, fork/pulls welcome if you want it sooner than I can get to it :)

**Comment by leetrout** [comment]
Under development in my fork in the "docs" branch. WIP at http://readthedocs.org/docs/django-zebra-doc-dev/
